code/src/components/Form.js

```diff
@@ -0,0 +1,238 @@
+import React, { useState } from 'react';
+import InputSelect from './InputSelect';
+import InputText from './InputText';
+import InputRadio from './InputRadio';
+import InputCheckbox from './InputCheckbox';
+import InputRange from './InputRange';
+import InputEmail from './InputEmail';
+import Button from './Button';
+import Summary from './Summary';
+import ProgressBar from './ProgressBar';
+import '../Styles/Form.css';
+
+export const Form = () => {
+	const [question, setQuestion] = useState(-1);
+	const [name, setName] = useState('');
+	const [season, setSeason] = useState('');
+	const [months, setMonths] = useState([]);
+	const [value, setValue] = useState(0);
+	const [age, setAge] = useState(false);
+	const [email, setEmail] = useState('');
+	const [submit, setSubmit] = useState(false);
+
+	const nextQuestion = () => setQuestion(question + 1);
+	const previousQuestion = () => setQuestion(question - 1);
+
+	const handleSubmit = event => {
+		event.preventDefault();
+		setSubmit(true);
+	};
+
+	const handleMonthsChange = monthValue => {
+		months.includes(monthValue)
+			? setMonths(months.filter(item => item !== monthValue))
+			: setMonths([...months, monthValue]);
+	};
+
+	//array for radio btn
+	const ageGroups = ['0-18 times', '19-40 times', '41-60 times', '61 + times'];
+
+	//array for checkbox
+	const seasonArray = [
+		{
+			season: 'Winter',
+			months: ['December', 'January', 'February'],
+		},
+		{
+			season: 'Spring',
+			months: ['March', 'April', 'May'],
+		},
+		{
+			season: 'Summer',
+			months: ['June', 'July', 'August'],
+		},
+		{
+			season: 'Fall',
+			months: ['September', 'October', 'November'],
+		},
+	];
+
+	return (
```

Comment:
I'm wondering if you have the correct indentation. You should check that your tab-values are set to 2. It looks like you have the setting on 4.

Reply:
No, I thought I had solve that, because in vs code everything looked ok. But I hope I have managed to do other settings now, hopefully for future projects it will look better and be easier to read on github 😄